cypress/support/commands.js

```diff
@@ -110,15 +114,45 @@ Cypress.Commands.add('createPin', () => {
   }
   cy.wait(2000)
 })
+
 Cypress.Commands.add('submitApplication', () => {
   // PREVIEW
   cy.get('#submit_form').click()
   // MODAL
   cy.get('#submit_confirm').click()
+  
   cy.log('Waiting for application to sync...')
+  cy.wait(10000)
+  cy.clock()
   cy.tick(40000)
-  cy.get('#row_0 #submitted0').should('exist')
+  //cy.get('#row_0 #submitted0').should('exist')
```

Comment:
@azantor64 by commenting this out you are not testing that the registration successfully submitted and appeared in the workqueue